### PR TITLE
Update the TOML

### PR DESCRIPTION
**Pull request by mzhong1**
update the toml with the latest crate versions and edit the code to use the updated crates

### CODE REVIEW

File: src/disk_manager.rs [hunk 1]
```diff
@@ -246,9 +246,9 @@ fn listen(
 
 fn respond_to_client<T: protobuf::Message>(result: &T, s: &mut Socket) -> BynarResult<()> {
     let encoded = result.write_to_bytes()?;
-    let msg = Message::from_slice(&encoded)?;
-    debug!("Responding to client with msg len: {}", msg.len());
-    s.send_msg(msg, 0)?;
+    //let msg = Message::from_slice(&encoded)?;
```
Comment (cholcombe973) [inline src/disk_manager.rs]: If these commented lines aren't needed anymore than they should be deleted.

File: src/backend/ceph.rs [hunk 2]
```diff
@@ -33,7 +32,13 @@ use tempdir::TempDir;
 
 /// Ceph cluster
 pub struct CephBackend {
-    cluster_handle: Rados,
+    /*
+        Note: RADONS (Reliable Autonomic Distributed Object Store)
```
Comment (cholcombe973) [inline src/backend/ceph.rs]: I think this should be `RADOS`?

File: src/backend/ceph.rs [hunk 3]
```diff
@@ -511,14 +517,14 @@ impl CephBackend {
...
-                let parts: Vec<String> = tag.split('=').map(|s| s.to_string()).collect();
+                let parts: Vec<String> = tag.split('=').map(ToString::to_string).collect();
                 if let Some(s) = parts.get(1) {
                     osd_id = Some(u64::from_str(s)?);
                 }
             }
             let fsid_tag = tags.iter().find(|t| t.starts_with("ceph.osd_fsid"));
             if let Some(tag) = fsid_tag {
-                let parts: Vec<String> = tag.split('=').map(|s| s.to_string()).collect();
+                let parts: Vec<String> = tag.split('=').map(ToString::to_string).collect();
```
Comment (cholcombe973) [inline src/backend/ceph.rs]: Interesting.  I didn't know you could do this.

File: src/backend/ceph.rs (hunk 2 above)
Comment (mzhong1) [reply src/backend/ceph.rs]: Lol whoops I better go fix those

File: src/backend/ceph.rs (hunk 3 above)
Comment (mzhong1) [reply src/backend/ceph.rs]: Clippy!

File: src/disk_manager.rs (hunk 1 above)
Comment (mzhong1) [reply src/disk_manager.rs]: Fixed!

**Pull request merged**